src/tokens/aoTokens/ao.ts

```diff
@@ -2,8 +2,7 @@ import { connect, dryrun } from "@permaweb/aoconnect";
 import { type Tag } from "arweave/web/lib/transaction";
 import { PersistentStorage } from "~utils/storage";
 import { Quantity } from "ao-tokens";
-import Arweave from "arweave";
-import { ArweaveSigner, createData } from "arbundles";
+import { Arweave, ArweaveSigner, createData } from "@dha-team/arbundles";
```

Comment:
Arweave must be imported from `arweave` package itself as importing from `@dha-team/arbundles` causes Arweave not being a constructor issue in Wander Connect.

```suggestion
import Arweave from "arweave";
import { ArweaveSigner, createData } from "@dha-team/arbundles";
```

Reply:
Added this change in my last commit.